packages/KTable/KTable.vue

```diff
@@ -80,6 +97,11 @@ table.k-table {
 .k-table {
   width: 100%;
   max-width: 100%;
+
+  &.sortable th {
```

Comment:
looks like the class is on the table, so every single column has a pointer cursor. We should define a prop "sortable" on `options.headers` objects that tells ktable if a given column should be clickable (i.e. cursor: pointer) and if it should emit the sort event.

Reply:
I'm confused. a table only has `sortable` if a sortKey is passed in. Yes every column is clickable if that happens but that's how it should be right? is there a situation where we would want only some columns to be sortable but others not? I thought it was either everything is sortable or everything isn't sortable.